server/index.ts

```diff
@@ -189,6 +193,69 @@ router.get('/:space/proposal/:id/votes', async (req, res) => {
     .then(obj => res.json(obj));
 });
 
+router.post('/:space/offchain_proofs', async (req, res) => {
+  const space = req.params.space;
+  const verifyingContract = req.body.verifyingContract;
+  const actionId = req.body.actionId;
+  const chainId = req.body.chainId;
+  const merkleRoot = req.body.merkleRoot;
+  const steps = req.body.steps;
+
+  if (
+    !space ||
+    !verifyingContract ||
+    !actionId ||
+    !chainId ||
+    !merkleRoot ||
+    !steps ||
+    steps.length < 1 // must have at least 2 steps
+  )
+    return res.status(400).send({
+      error: 'invalid request parameters'
+    });
+
+  const indexes = steps.map(s => s.index).filter(s => s.index);
```

Comment:
Are `s.index` a `number`? If so, `0` will not pass the `filter`. E.g.:

```js
[{index: 0}].map(s => s.index).filter(s => s.index)

[]
```

Reply:
great catch. Thanks. Just fixed that.